spec/support/shared_examples/an_email_address_valid_for_notify_spec.rb

```diff
@@ -0,0 +1,73 @@
+RSpec.shared_examples 'an email address valid for notify' do |field_name = :email_address|
+  [
+    # Standard email addresses
+    'user@example.com',                              # Standard format
+    'user123@example.com',                           # With numbers in local part
+    'user.name@example.com',                         # With period in local part
+    'user_name@example.com',                         # With underscore in local part
+    'user+tag@example.com',                          # With plus symbol in local part
+    'user-tag@example.com',                          # With hyphen in local part
+    'user@hotmail.co.uk',
+    'user@yahoo.co.uk',
+    'user@myportabledomain.ab.uk',
+
+    # Email addresses with subdomains
+    'user123@subdomain.example.com',                 # With subdomain and numbers in local part
+    'user.name@sub-domain.example.com',              # With subdomain and period in local part
+    'user+tag@subdomain.example.com',                # With subdomain and plus symbol in local part
+    'user-tag@subdomain.example.com',                # With subdomain and hyphen in local part
+    'john_doe@sales.example.com',                    # With nested subdomains
+
+    # International local part
+    '阿里巴巴@example.com', # Chinese
+    'サンプル@example.com', # Japanese
+
+    # Email addresses with Internationalized Domain Names (IDNs)
+    'user@xn--bcher-kva.example.com',               # With IDN (contains special characters)
+    'user@xn--5hx875a.example.com',                 # With IDN (contains Chinese characters)
+    'user@sub.xn--bcher-kva.example.com',           # With subdomain and IDN
+    'user@sub.xn--5hx875a.example.com',             # With subdomain and IDN
+
+    # Email addresses with different TLDs
+    'user@example.co.uk',                            # With UK TLD
+    'user@sub.example.co.uk',                        # With subdomain and UK TLD
```

Comment:
This is the same as this line, no?
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9134/files#diff-8f9a4aca678e3d43f1dacd632ee0dbd65635f020d500b1bc9ffe1d88f8cbb53dR36

Reply:
True! I will remove

Reply:
Done removed!